gravitee-apim-rest-api/gravitee-apim-rest-api-model/src/main/java/io/gravitee/rest/api/model/UpdateApplicationEntity.java

```diff
@@ -76,7 +76,7 @@
     private boolean disableMembershipNotifications;
 
     @JsonProperty("api_key_mode")
-    @ApiModelProperty(value = "The API key mode used for this application.", allowableValues = "SHARED, EXCLUSIVE")
+    @ApiModelProperty(value = "The API key mode used for this application.", allowableValues = "UNSPECIFIED, SHARED, EXCLUSIVE")
```

Comment:
I am wondering if we should state in the spec that `UNSPECIFIED` is an acceptable update value for the key mode ?

Do we expect API clients to explicitly set the value to `UNSPECIFIED` by issuing a `PUT` ?